gradle.properties

```diff
@@ -3,12 +3,12 @@
 kotlin.code.style=official
 org.gradle.jvmargs=-Xmx3G
 # Fabric
-minecraft_version=1.16.3
-yarn_mappings=1.16.3+build.7
-loader_version=0.9.3+build.207
-loom_version=0.4-SNAPSHOT
+minecraft_version=1.16.4
+yarn_mappings=1.16.4+build.1:v2
+loader_version=0.10.6+build.214
```

Comment:
Do we _need_ to bump the loader version? I'm probably wrong, but I thought it was backwards/forward compatible

Reply:
I have no idea, I didn't test without